src/test/java/org/folio/rest/api/CallNumberNormalizedTestBase.java

```diff
@@ -18,103 +18,11 @@
 
   protected abstract List<String> searchByCallNumberNormalized(String callNumber) throws Exception;
 
-  @Parameters({
```

Comment:
The three classes CallNumberNormalizedTestBase, HoldingsCallNumberNormalizedTest.java, ItemCallNumberNormalizedTest can be completely deleted. The remaining tests methods cannotSearchByNonAlphanumericsOnly and cannotSearchBySuffixOnly don't server any purpose any longer.